### PR TITLE
Created an ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by DatscIY**
Issue #108 

Created a .github/ISSUE_TEMPLATE subdirectory and created an issue_template.md file with issue criteria.

### CODE REVIEW

**Comment by Arfey** [comment]
Thx for your contribution :)

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+##Steps to Reproduce the Problem
+1.
+2.
+3.
+
+##Specifications
+- Version:
+- Platform:
+- Subsystem:
```
Comment (Arfey) [inline .github/ISSUE_TEMPLATE/issue_template.md]: I think that will be great to describe in comments what user need to write in each section.

This looks some like that

```
## Long story short

<!-- Please describe your problem and why the fix is important. -->
```

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Expected Behavior
+
+##Actual Behavior
+
+##Steps to Reproduce the Problem
+1.
+2.
+3.
+
+##Specifications
```
Comment (Arfey) [inline .github/ISSUE_TEMPLATE/issue_template.md]: In my opinion `Platform` and `Subsystem` are not helpful information. Instead that we can ask about version  of `python` / `docker`.

Comment (DatscIY) [reply .github/ISSUE_TEMPLATE/issue_template.md]: added requested changes. Thanks.

**Comment by DatscIY** [comment]
Closes #108

**Pull request closed without merging**